### PR TITLE
Replace MEILISEARCH_HOST by MEILISEARCH_URL

### LINKED ISSUE #390: Replace MEILISEARCH_HOST by MEILISEARCH_URL in the docker config
⚠️ This issue is generated, it means the examples and the namings do not necessarily correspond to the language of this repository.
Also, if you are a maintainer, feel free to add any clarification and instruction about this issue.

Sorry if this is already partially/completely implemented, feel free to let me know about the state of this issue in the repo.

Related to https://github.com/meilisearch/integration-guides/issues/214

----

Replace the env var naming from the Dockerfile environment:

- `MEILISEARCH_HOST` to `MEILISEARCH_URL`

Also, look for occurrences inside of the docker-compose.yml file.

After, run the tests according to the instructions in the CONTRIBUTING.md.

Also, check for occurences in the Github Actions workflows and inside of the source code.

### PR DESCRIPTION
**Pull request by girijakar**
# Pull Request

## Related issue
Fixes #390 

## What does this PR do?
- It replace MEILISEARCH_HOST by MEILISEARCH_URL in the docker config

## PR checklist
Please check if your PR fulfills the following requirements:
- [X] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [X] Have you read the contributing guidelines?
- [X] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ services:
     stdin_open: true
     working_dir: /home/package
     environment:
-      - MEILISEARCH_HOST=http://meilisearch:7700
+      - MEILISEARCH_URL=http://meilisearch:7700
```
Comment (brunoocasali) [inline docker-compose.yml:10]: You still have to use the new env var, otherwise, the test will not load the configuration.

**Review by brunoocasali — CHANGES_REQUESTED** [review]
Hi @girijakar please check the log of the action, there are still some places to be changed :)

**Comment by girijakar** [comment]
Ok, I am working on it

**Comment by girijakar** [comment]
> Hi @girijakar please check the log of the action, there are still some places to be changed :)

Hi, @brunoocasali, my doubt may seem a bit childish, please bear with me. I am a bit new to OS.
Can you please point me where the changes are required more, I mean I rechecked all the code files and those 2 are the only files which have the HOST variable?

File: docker-compose.yml (hunk 1 above)
Comment (brunoocasali) [reply docker-compose.yml:10]: Not really. The problem is this: https://github.com/meilisearch/meilisearch-php/blob/4ed1dca16e936caac9ac8e087d9bb794a436a793/tests/TestCase.php#L50 You have to change in the TestCase.php as well :)

Comment (girijakar) [reply docker-compose.yml:10]: Updated, thank you.

**Comment by brunoocasali** [comment]
_This message is sent automatically_

Thanks again for contributing to Meilisearch :heart:
If you are participating in [Hacktoberfest](https://blog.meilisearch.com/contribute-hacktoberfest-2022/), and you would like to receive some gift from Meilisearch too, please complete [this form](https://meilisearch.typeform.com/to/idt5qugY).

**Pull request merged**